seahorn/gatekeeper/CMakeLists.txt

```diff
@@ -0,0 +1,3 @@
+
+add_subdirectory(stubs)
```

Comment:
PR includes file `.DS_Store`. I cannot comment on that file so commenting here. Please remove it.